system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -731,13 +731,67 @@
                               :Name "unicode-test Großartiger Scott!"
                               :CollectionConceptId (:concept-id coll)})
                             :native-id)
-            {:keys [native-id concept-id status]} (ingest/ingest-concept concept {:token token
-                                                                                  :method :post})]
+            {:keys [status concept-id native-id revision-id]}
+            (ingest/ingest-concept concept
+                                   {:token token
+                                    :method :post})]
         (is (= 201 status))
         (is (not (nil? concept-id)))
-        (is (not (nil? native-id)))
         (is (string/starts-with? native-id "unicode_test_großartiger"))
+        (is (= 1 revision-id))
 
         (index/wait-until-indexed)
 
         (is (not (nil? (:native-id (first (:items (subscription-util/search-json {:name (:Name concept)})))))))))))
+
+(deftest create-subscription-by-put
```

Comment:
Why don't we have tests for POST?

Reply:
The tests for POST are in the test above this one.